internal/models/deployment_detail.go

```diff
@@ -0,0 +1,43 @@
+package models
+
+import (
+	"time"
+)
+
+type DeploymentDetail struct {
```

Comment:
Open to suggestions on this name. I didn't want to call it `Deployment`, since it has a `Deployment` field.